cmd/grype/cli/options/grype.go

```diff
@@ -34,7 +34,8 @@ type Grype struct {
 	Name                   string             `yaml:"name" json:"name" mapstructure:"name"`
 	DefaultImagePullSource string             `yaml:"default-image-pull-source" json:"default-image-pull-source" mapstructure:"default-image-pull-source"`
 	VexDocuments           []string           `yaml:"vex-documents" json:"vex-documents" mapstructure:"vex-documents"`
-	VexAdd                 []string           `yaml:"vex-add" json:"vex-add" mapstructure:"vex-add"` // GRYPE_VEX_ADD
+	VexAdd                 []string           `yaml:"vex-add" json:"vex-add" mapstructure:"vex-add"`                                     // GRYPE_VEX_ADD
+	ShowKernelHeaders      bool               `yaml:"show-kernel-headers" json:"show-kernel-headers" mapstructure:"show-kernel-headers"` // Show matches on kernel-headers packages where the match is on kernel upstream instead of marking them as ignored, default=false
```

Comment:
Maybe this should be a little more specific like `include-kernel-headers-with-kernel-upstream` or similar, what do you think?

Reply:
yeah, I agree with specificity, just gets very verbose. I'm thinking "include-upstream-kernel-matches-on-headers"? which is a mouthful, but if not a command-line option, then may be not too bad.

Reply:
I went with: `show-kernel-headers-matched-upstream-kernel`. Good, or do you prefer 'include' to 'show'? I was used 'show' to indicate the matches are present just ignored vs not included at all, but I wasn't sure about the precedent for that. I only saw the `show-suppressed` option that was similar. I'm not strongly opinionated on this, so if 'include' is better, I'll change it.